roles/openjdk/molecule/default/verify.yml

```diff
@@ -16,3 +16,4 @@
       ansible.builtin.assert:
         that:
           - not pkg_status.changed
+
```

Comment:
This one made me laugh. Thanks, linter!

As Java is required for Solr, this change makes sense in this PR.